src/components/EditorWidgets/Relation/RelationControl.js

```diff
@@ -90,7 +90,14 @@ class RelationControl extends Component {
 
   renderSuggestion = (suggestion) => {
     const { field } = this.props;
-    const valueField = field.get('valueField');
+    const valueField = field.get('displayFields').toJS() || field.get('valueField');
```

Comment:
This would error out if `displayFields` isn't a list. Instead of converting to JS, just leave the value as is and switch your `Array.isArray` check to [`List.isList`](https://facebook.github.io/immutable-js/docs/#/List/isList).

Reply:
Updated